client/js/components/data_page.jsx

```diff
@@ -60,33 +66,37 @@ var TableHeadersRow = React.createClass({
               return <th>{header}</th>;
             })
           }
+          <th>Download</th>
         </tr>
       );
     }
 });
 
 var Table = React.createClass({
+ componentDidMount: function(){
+  AppStore.renderSchema(this.props.schemaName);
+ },
  getInitialState: function() {
-  return {
-    mockFitBitData: [
-      {name: "Brian", miles: 1, month: "feb"},
-      {name: "Thomas", miles: 3.4, month: "march"},
-      {name: "Zindler", miles: 45, month: "july"},
-      {name: "Murphy", miles: 2, month: "jan"}
-    ]
-  };
+  return {};
  },
  render: function() {
    var Table = reactBootStrap.Table;
-   var fitBitData = this.state.mockFitBitData;
-   var headerKeys = Object.keys(fitBitData[0]);
-
-   return (
-     <Table striped bordered condensed hover>
-        <TableHeadersRow headerKeys={headerKeys}/>
-        <TableRows rowData={fitBitData}/>
-     </Table>
-   );
+   
+   if(this.props.displaySchema){
+     var rows = this.props.displaySchema;
```

Comment:
do this before the `if` statement. also, do you need to check if `this.props` exists?